client/src/pages/Message/MessageWindow/TextContent/TextContent.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box } from '@material-ui/core';
+import { Conversation } from '../../../../interface/Message';
+import OwnerBubble from '../OwnerBubble/OwnerBubble';
+import UserBubble from '../UserBubble/UserBubble';
+import useStyles from './useStyles';
+
+export interface MessageProps {
+  text?: Conversation;
+}
+
+export default function TextContent({ text }: MessageProps): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Box className={classes.root}>
+      {text?.messages.map((message) => {
```

Comment:
When we map over an array of items, we want each individual item to have a unique key.

Ref: https://dev.to/francodalessio/understanding-the-importance-of-the-key-prop-in-react-3ag7

Reply:
thank you for the ref website, and I add key as messageId